### ISSUE TITLE
LDC 1.2.0-beta1 SIMD regression

### ISSUE THREAD

**Issue by 9il** [ldc-developers/ldc#2056]
This code works in recent DMD and LDC <=1.1.x
```d
void load(ref __vector(double[2]) a, double b)
{
    a = b;
}
```

LDC 1.2.0 output:
```
constant expression expected, not b
```

This feature is used in GLAS.

**Comment by kinke** [comment]
Hmm, compiles fine with master on Win64 (for Linux too), but I can reproduce it with 1.2.0-beta1.

**Comment by kinke** [comment]
Thanks for the report, fixed in branch `release-1.2.x` and will be in the next beta.
